src/pages/timelines.md

```diff
@@ -8,10 +8,16 @@ Please have a look at [Release management](/release-management), to get more inf
 
 ![Overall timeline for Releases from 24.12 to 25.12 ](@site/static/img/cx-timeline-overview.png)
 
-## Catena-X Io (Preview), incl. Eclipse Tractus-X Release 24.12
+## Overview of Catena-X and Eclipse Tractus Release Schedules
+
+Below are the current timelines for the upcoming releases.
+Milestones marked with the Tractus-X logo are open to all and can be accessed through the [Tractus-X Open Meetings page](https://eclipse-tractusx.github.io/community/open-meetings).
+Milestones featuring the Catena-X logo indicate closed meetings, which are organized by invitation only and facilitated by the Catena-X Association.
+
+### Catena-X Io (Preview), incl. Eclipse Tractus-X Release 24.12
 
 ![Timeline for Release 24.12 ](@site/static/img/cx-timeline-24.12.png)
 
-## Catena-X Io, incl. Eclipse Tractus-X Release 25.03
+### Catena-X Io, incl. Eclipse Tractus-X Release 25.03
```

Comment:
Is this correct? Shouldn't it be including 24.12 and 25,03?

Reply:
No I don't think so, because it is only the timeline for the 25.03. If we would write 24.12 and 25.03 we would have to show the timeline of 24.12 with it or not? 
But right now I not sure either.

Reply:
Mh not sure :) Eclipse Tractus-X Release 25.03 means for me the released artefacts, the new functionality … and what is included in Io? Functionality of 24.12 and 25.03 ... for me, it's the same as for the Ecosystem Release field on the Roadmap Items... so I think @werner-roman has to decide. :)

Reply:
Hey :)

well pick the definition of the terms "preview" etc up in the DSOC as prior.
Made a comitt which circumvents the issue as of now :)